discovery-provider/src/queries/get_health.py

```diff
@@ -328,6 +336,9 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
         "latest_block_num": latest_block_num,
         "latest_indexed_block_num": latest_indexed_block_num,
         "final_poa_block": final_poa_block,
+        "network": {
+            "discoveryNodes": discovery_nodes
```

Comment:
typo in my doc probs, but since all of our stuff is snake_case, probably should be the same here